search/intermediate_result.cpp

```diff
@@ -291,6 +291,9 @@ void FillDetails(FeatureType & ft, Result::Details & details)
 
   auto const roadShields = ftypes::GetRoadShieldsNames(ft);
   details.m_roadShields = strings::JoinStrings(roadShields, osm::MapObject::kFieldsSeparator);
+    
+  auto const fee = feature::GetLocalizedFeeType(feature::TypesHolder(ft));
```

Comment:
nit: would be good to init the TypesHolder once only and then re-use for all checks like GetLocalizedCuisines, IsHotelChecker...
(`IsHotelChecker::Instance()(ft);` inits a TypesHolder under the hood)

Reply:
Done